api/api_test.go

```diff
@@ -61,6 +61,11 @@ func makeClientWithConfig(
 	var server *testutil.TestServer
 	var err error
 	retry.RunWith(retry.ThreeTimes(), t, func(r *retry.R) {
+		// Skip test when -short flag provided; any tests that create a test
+		// server will take at least 100ms which is undesirable for -short
+		if testing.Short() {
+			t.Skip("too slow for testing.Short")
+		}
```

Comment:
Generally we put these as the first line to avoid doing any work if the test will be skipped. Here it's in a retry. I think it might be better placed as the first line of `makeClientWithConfig`.